streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/TopologyTest.scala

```diff
@@ -101,14 +99,14 @@ class TopologyTest extends JUnitSuite {
       val _: KTable[String, Long] =
         textLines
           .flatMapValues(v => pattern.split(v.toLowerCase))
-          .groupBy((k, v) => v)
+          .groupBy((_, v) => v)
```

Comment:
This was an IDEA suggestion to use an `_` instead of a name for parameters that are unused. It's a form of self-documenting code.